KnowWhereGraph/kwg-faceted-search
# Expert Querying

This PR fixes issue #282 and is the first of three tabs to have the query functionality completed.

To test:
1. Check out
2. Generate cache if needed
3. Start the local deployment
4. Navigate to the Peoples tab and make sure that the results look correct
5. When no results are found, note that the count now says "0 Results" rather than "At Least 0 Results"
